jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/security/authentication/FormAuthenticator.java

```diff
@@ -223,12 +223,12 @@ public Authentication validateRequest(Request req, Response res, Callback callba
         ServletContextRequest servletContextRequest = Request.as(req, ServletContextRequest.class);
         ServletContextRequest.ServletApiRequest servletApiRequest = servletContextRequest.getServletApiRequest();
 
-        boolean jSecurityCheck = isJSecurityCheck(req.getPathInContext());
+        boolean jSecurityCheck = isJSecurityCheck(Request.getPathInContext(req));
         mandatory |= jSecurityCheck;
         if (!mandatory)
             return new DeferredAuthentication(this);
 
-        if (isLoginOrErrorPage(req.getPathInContext()) && !DeferredAuthentication.isDeferred(res))
+        if (isLoginOrErrorPage(Request.getPathInContext(req)) && !DeferredAuthentication.isDeferred(res))
```

Comment:
Store `Request.getPathInContext(request)` in a local variable to avoid calling it twice.